tests/__snapshots__/default.test.ts.snap

```diff
@@ -1,19 +1,140 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`Empty default 1`] = `
+exports[`Can't convert double quote @default if a single quote character is in the string 1`] = `
 "/**
- * The value
+ * The summary
  *
- * @default 'type' name description
+ * @default \\"This isn't bad\\"
  */
 "
 `;
 
-exports[`Empty default 2`] = `
+exports[`convert double quote @default to single quote 1`] = `
 "/**
- * The value
+ * The summary
  *
- * @default
+ * @default 'value'
+ */
+"
+`;
+
+exports[`convert single quote @default to double quote 1`] = `
+"/**
+ * The summary
+ *
+ * @default \\"value\\"
+ */
+"
+`;
+
+exports[`default empty array 1`] = `
+"/**
+ * The summary
+ *
+ * @default [ ]
+ */
+"
+`;
+
+exports[`default empty array in square-bracket boilerplate 1`] = `
+"/**
+ * The summary
+ *
+ * @default [[]]
+ */
+"
+`;
+
+exports[`default empty object 1`] = `
+"/**
+ * The summary
+ *
+ * @default { }
+ */
+"
+`;
+
+exports[`default empty object in square-bracket boilerplate 1`] = `
+"/**
+ * The summary
+ *
+ * @default [{}]
+ */
+"
+`;
+
+exports[`default filled array 1`] = `
+"/**
+ * The summary
+ *
+ * @default [1, \\"two\\", { three: true }, [\\"four\\"]]
+ */
+"
+`;
+
+exports[`default filled object 1`] = `
+"/**
+ * The summary
+ *
+ * @default { object: \\"value\\", nestingTest: { obj: \\"nested\\" } }
+ */
+"
+`;
+
+exports[`default string with description 1`] = `
+"/**
+ * The summary
+ *
+ * @default \\"type\\" description
+ */
+"
+`;
+
+exports[`default value empty array 1`] = `
+"/**
+ * The summary
+ *
+ * @defaultValue [ ]
```

Comment:
Oh, this is actually wrong. Based on the documentation `@defaultValue` is supposed to be all lowercase (`@defaultvalue`)

Reply:
An all lowercase `@defaultvalue` tag was more difficult to implement than I expected. I'm dropping this as being out of scope for this PR.